common/src/test/java/org/broadleafcommerce/common/sitemap/service/SiteMapGeneratorTest.java

```diff
@@ -147,7 +153,7 @@ protected String convertFileToString(File file) throws IOException {
             if (line.contains("</lastmod>")) {
                 continue;
             }
-            line = line.replaceAll("\\s+", "");
+            line = line.replaceAll("\\s+", " ");
```

Comment:
Is this change necessary to fix the flaky test?

Reply:
Yes it is required, XML files are compared based on their structure.